tgui/packages/tgui/styles/interfaces/Playtime.scss

```
@@ -0,0 +1,35 @@
+@use '../base.scss';
+
+.PlaytimeInterface {
+  td {
+    padding: 0;
+    height: 34px;
+  }
+  table {
+    border: 1px solid white;
+    border-radius: 4px;
+  }
+
+  tr {
+    border-bottom: 1px solid white;
+  }
```

```suggestion
  .Table__cell {
    border-bottom: 1px solid white !important;
  }
```
the issue with using the hive_status theme is that hivestatus.scss contains .Table__cell setting the bottom border to 1px, which makes the bottom of the xeno ranks look pretty goofy. this overrides the theme for this interface. kinda hacky but what can you do